tests/frontend/critical_designs/base_procedure.py

```diff
@@ -1,17 +1,17 @@
-import time
-import logging
+import unittest
 
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 
-from frontend import uiUtils
-from frontend import webdriverUtils
-from frontend import gcodeUtils
-from frontend import frontendTestUtils
+from .. import uiUtils
+from .. import webdriverUtils
+from .. import gcodeUtils
+from .. import frontendTestUtils
```

Comment:
I don't like these .. does this work too?

```suggestion
from tests.frontend import webdriverUtils, gcodeUtils, frontendTestUtils, uiUtils
```

Reply:
I changed all the relative imports to absolute